encryption_works.md

```diff
@@ -120,103 +120,100 @@ When Snowden [was answering questions on Guardian's website](http://www.guardian
 
 ## Off-the-Record (OTR) Chat
 
-[Off-the-Record](http://www.cypherpunks.ca/otr/) (OTR) is a layer of encryption that can be added to any existing instant message chat system, provided that you can connect to that chat system using a chat client that supports OTR, such as [Pidgin](https://pidgin.im/) or [Adium](http://adium.im/). With OTR it's possible to have secure, end-to-end encrypted conversations over services like Google Talk and Facebook chat without Google or Facebook ever having access to the contents of the conversations. Note: this is different than the "off-the-record" option in Google, which is **not** secure. And remember: while Google and Facebook’s HTTPS connection is very valuable for protection against your message while it’s in transit, they still have the keys to your conversations so they can hand them over to authorities.
+The best way to end-to-end encrypt your instant messages is to use something called OTR chat. You can use OTR with both Google and Facebook, or you can set up a Jabber account, which could provide you with more anonymity. Using OTR allows you to use secure chats over Google Talk or Facebook chat without Google or Facebook ever being able to see the content of your conversations.
```

Comment:
It's not tied to the underlying provider. You could have an OTR chat with carrier pigeons if you wanted to.

We need to strike a balance with this sentence. While it is true (and often convenient) that you can use OTR with popular chat solutions like Google Talk or Facebook Chat, it is also possible (and almost always better for privacy) to use an XMPP server such as the ones provide by Calyx Institute or Duck Duck Go.

I understand that as the section progresses, many of these things are explained and recommend. Still, I think we can do better with this first paragraph.